zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProvider.java

```diff
@@ -101,8 +100,8 @@ public class PrometheusMetricsProvider implements MetricsProvider {
     private final CollectorRegistry collectorRegistry = CollectorRegistry.defaultRegistry;
     private final RateLogger rateLogger = new RateLogger(LOG, 60 * 1000);
     private String host = "0.0.0.0";
-    private int port = 7000;
-    private boolean exportJvmInfo = true;
+    protected int port = 7000;
```

Comment:
I would add support for a secondary port here instead of creating an alternative metrics provider.

The Metrics Provider is the library/system that you use to export metrics, and it should not be related to security/https

Can you please merge the two classes in one ?